### DISCUSSION TITLE
Better drag-drop approach to Data model and new Folder feature

### DISCUSSION THREAD

**Discussion by darioguarascio** [directus/directus#8862] [Feature Requests]
Folders in data model is a very cool feature when it comes to navigate collections, but a bit of a nightmare when it comes to create and manage folders. I have some tens collections and i had to drag them up and down to put them in appropriate folders.
Perhaps a collapsible option in data model too would help shorten the distance of dragging?

**Comment by rijkvanzanten** [comment]
We might be able to show the "group" field on the collections detail page, and then use the system-collections interface to choose the "parent group". We might want to update that system-collection interface to show the collections nested, for ease of selection 🤔

**Reply by benhaynes** [reply]
Kind of like how we have a "Move To" button on the File Detail page?

**Reply by darioguarascio** [reply]
@benhaynes `Move to` it is surely an option, but if the destination list is a big one, the problem comes back. I also like the related collection selection, a simple autocomplete.

**Comment by lukas-schaetzle** [comment]
I remember that there was a preview video in the pull request for the new data model setup where this feature (collapsible folders) was already present. Not sure why that didn't make it in. I think it would be a good solution.

Also, I had some trouble dropping the items into the right level: For example, I just want to move a 1st level item below some 2nd level items (but keep it on 1st level). Then most of the time the interface wants to place it into the child item hierarchy from above (so in 2nd or even 3rd level)

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
